pygmt/src/sphdistance.py

```diff
@@ -35,6 +35,12 @@ def sphdistance(table, **kwargs):
 
     Parameters
```

Comment:
Can't make a suggested change here, but could you include a line:

```
Full option list at :gmt-docs:`sphdistance.html
```
above the `{aliases}` line? Thanks!